mkt/websites/models.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from django.conf import settings
+from django.db import models
+
+import json_field
+
+from mkt.site.models import ModelBase
+from mkt.translations.fields import TranslatedField
+from mkt.tags.models import Tag
+from mkt.translations.fields import save_signal
+
+
+class Website(ModelBase):
+    default_locale = models.CharField(max_length=10,
+                                      default=settings.LANGUAGE_CODE)
+    url = TranslatedField()
```

Comment:
@chuckharmston pointed out that Googles use `<link rel="alternate" hreflang="xx" href="yy" />` so we could detect this automatically, and if this `link` tag is found, we'd take the canonical url and extract the various locale versions automatically, and never let the submitter to it themselves.